### PR TITLE
feat(export): Map entities are now exported with map

### PR DESCRIPTION
**Pull request by DeltaDesigns**
- Map entities are now exported with map
- Snuck in a fix for entity mesh normals being broke
- Fix map fbx always exporting as arranged

### CODE REVIEW

File: Tiger/Schema/Entity/Entity.cs [hunk 1]
```diff
@@ -34,6 +34,8 @@ public Entity(FileHash hash, bool shouldParse = true) : base(hash, shouldParse)
     public void Load()
     {
         Deserialize();
+        if (_tag.FileSize == 0)
```
Comment (MontagueM) [inline Tiger/Schema/Entity/Entity.cs:37]: why is this needed?

Comment (DeltaDesigns) [reply Tiger/Schema/Entity/Entity.cs:37]: some entities were triggering the Debug.Assert for some reason so that was my 'quick fix'

File: Charm/ActivityMapView.xaml.cs [hunk 2]
```diff
@@ -222,6 +223,8 @@ public async void ExportFull(ExportInfo info)
         // MainWindow.Progress.CompleteStage();
 
         Tiger.Exporters.Exporter.Get().Export();
+        Tiger.Exporters.Exporter.Get().Reset(); //Clear the scenes for the next export
```
Comment (MontagueM) [inline Charm/ActivityMapView.xaml.cs:226]: can you add it so that the Export() function clears the scenes? would be annoying to have to do this after every export

File: Tiger/Schema/Entity/Entity.cs (hunk 1 above)
Comment (MontagueM) [reply Tiger/Schema/Entity/Entity.cs:37]: debug assert is only triggered in a debug build; its there to alert theres an issue (in this case, theres a bug somewhere in the deserialization)
you can fix while testing by commenting it out, but leave it in pls :))))))

File: Tiger/Exporters/Exporter.cs [hunk 3]
```diff
@@ -88,7 +92,7 @@ public void AddStaticInstancesAndParts(FileHash meshHash, List<StaticPart> parts
         }
         StaticMeshes.Add(mesh);
 
-        StaticMeshInstances.TryAdd(meshHash, InstancesToTransforms(instances));
```
Comment (MontagueM) [inline Tiger/Exporters/Exporter.cs:91]: is this needed? why does arranged need to be split?

Comment (DeltaDesigns) [reply Tiger/Exporters/Exporter.cs:91]: It was causing maps to export as arranged on non-arranged exports, which would break the blender importer

Comment (MontagueM) [reply Tiger/Exporters/Exporter.cs:91]: ahhhhhhh right im dumb thanks

File: Tiger/Schema/Entity/Entity.cs (hunk 1 above)
Comment (MontagueM) [reply Tiger/Schema/Entity/Entity.cs:37]: will approve after this is removed

Comment (DeltaDesigns) [reply Tiger/Schema/Entity/Entity.cs:37]: oh oops, should be removed now
